src/palace/manager/integration/license/opds/for_distributors/api.py

```diff
@@ -41,17 +37,13 @@
 from palace.manager.sqlalchemy.model.patron import Loan, Patron
 from palace.manager.sqlalchemy.model.resource import Hyperlink
 from palace.manager.sqlalchemy.util import get_one
-from palace.manager.util import base64
 from palace.manager.util.datetime_helpers import utc_now
-from palace.manager.util.http import HTTP
 
 
 class OPDSForDistributorsAPI(
     BaseCirculationAPI[OPDSForDistributorsSettings, OPDSForDistributorsLibrarySettings],
     HasCollectionSelfTests,
 ):
-    BEARER_TOKEN_CREDENTIAL_TYPE = "OPDS For Distributors Bearer Token"
```

Comment:
After this change, we don't store the bearer token credential in the database. The old credentials that are already stored should just expire and be removed normally, so no need for a migration to clean them up.